### PR TITLE
network/iptable: do not drop packets to 127/8

### PR DESCRIPTION
**Pull request by Neirpyc**
See #16 : this PR allows packets to be sent to `127.0.1.1` and more generally `127/8` instead of only `127.0.0.1` as in @Dettorer's original work.

### CODE REVIEW

**Comment by Dettorer** [comment]
Thank you for taking the lead on that and sorry for not reacting sooner.

I never saw that syntax `127/8`, did you try to build envlive with that and to use Pycharm's debugger with it? I mentioned in my old PR that I didn't test besides throwing in the command directly on some booted environment.

**Comment by Neirpyc** [comment]
I ran `sudo iptables -A OUTPUT -d 127/8 -j ACCEPT` which worked and `sudo iptables -A OUTPUT -d 127./8 -j ACCEPT` which failed. Therefore, this format should be valid.
I took the lead as I didn't think you'd still follow this PR half a year later, if you want, I can close this PR and you can add a commit to yours (you did the hardest part anyway, you deserve to keep your PR :).

**Comment by Dettorer** [comment]
> I ran (on my machine) `sudo iptables -A OUTPUT -d 127/8 -j ACCEPT` which worked and `sudo iptables -A OUTPUT -d 127./8 -j ACCEPT` which failed. Therefore, this format should be valid.

Ok great.

> I took the lead as I didn't think you'd still follow this PR half a year later, if you want, I can close this PR and you can add a commit to yours (you did the hardest part anyway, you deserve to keep your PR :).

Oh no really it's totally cool, it's even possible I would never have seen the ping by mail unless someone pinged me on the association's internal chat. You can add a `Co-authored-by:` in the commit message after you squash the PR if you want but that's really not necessary.

**Comment by Neirpyc** [comment]
> You can add a `Co-authored-by:` in the commit message after you squash the PR if you want.
I'm pretty new to github, I'd love to do this, but I have no idea how to squash two commits in an already open PR, If you walk me through, I could totally do this!

**Comment by rissson** [comment]
> I never saw that syntax `127/8`,

`127/8` is just a shortcut for `127.0.0.0/8`, writing IP addresses is long and I'm lazy :D

File: root_skel/etc/NetworkManager/dispatcher.d/pre-up.d/00_filter.sh [hunk 1]
```diff
@@ -7,4 +7,5 @@ subnet="$(ip -o a show dev "$device" scope global | awk '{ print $4 }')"
 iptables -F
 
 iptables -A OUTPUT -d "$subnet" -j ACCEPT
+iptables -A OUTPUT -d 127/8 -j ACCEPT
```
Comment (rissson) [inline root_skel/etc/NetworkManager/dispatcher.d/pre-up.d/00_filter.sh:10]:
```suggestion
iptables -A OUTPUT -d 127.0.0.0/8 -j ACCEPT
```

> I ran (on my machine) sudo iptables -A OUTPUT -d 127/8 -j ACCEPT which worked and sudo iptables -A OUTPUT -d 127./8 -j ACCEPT which failed. Therefore, this format should be valid.

Let's use the long format, just to be safe.

**Review by rissson — COMMENTED** [review]
.

**Comment by Elfikurr** [comment]
Hello,
Thanks a lot for your work :)
In order to squash the commits, you need to use the command
`git rebase -i`
It will prompt you with the list of your commits, and you can specify an action (squash here) on the a commit the squash it with the one above (the list of the actions is displayed when running the command).
You can then
`git commit --amend`
In order to amend the commit message (adding the `Co-authored-by`.
Then you can
`git push <your remote> <your branch> -f`
And you're all set !
Do not hesitate to ask for more details if needed :)

**Comment by Neirpyc** [comment]
@Elfikurr thanks for your help!
@Dettorer I think I made you the commit owner and me the Co-author, which is not what you suggested, but is actually closer to what really happened, I hope you don't mind!
@rissson Accepted your commit :)

**Review by rissson — APPROVED** [review]

**Pull request merged**